dabl-meta.yaml

```diff
@@ -1,17 +1,11 @@
 catalog:
     name: dabl-chat
-    version: 0.1.5
+    version: 0.2.0
     short_description: DABL Chat
     description: A straightforward web chat app.
-    release_date: 2020-09-30
```

Comment:
we no longer have a `release_date` field?

Reply:
It's still there, but now automatically populated by the build tool.  `ddit` will parse `dabl-meta.yaml`, make a few updates and checks of this sort, and then rewrites the result into the DIT file. (This is also why the file no longer explicitly contains subdeployment listings. You _can_ have them, but the tool looks at the command line arguments and DAR file build and includes those automatically.)